lib/simplifyExpression/collectAndCombineSearch/multiplyLikeTerms.js

```diff
@@ -157,7 +157,8 @@ function collectConstantExponents(node) {
   const baseNode = ConstantOrConstantPower.getBaseNode(node.args[0]);
   // The new exponent will be a sum of exponents (an operation, wrapped in
   // parens) e.g. 10^(3+4+5)
-  const exponentNodeList = node.args.map(p => ConstantOrConstantPower.getExponentNode(p));
+  const exponentNodeList = node.args.map(
```

Comment:
OH i just remembered you can do 

`node.args.map(ConstantOrConstantPower.getExponentNode)`

it's a special case when you just do `p => f(p)`, then you can just pass the function you're using `f` instead